platform/jobs/edxPlatformLettuceMaster.groovy

```diff
@@ -98,7 +98,9 @@ secretMap.each { jobConfigs ->
         }
 
         parameters {
-            stringParam(stringParams.name, stringParams.default, stringParams.description)
+            stringParams.each { param ->
```

Comment:
@estute do we know why this was working before and then it wasn't? E.g. have any versions of jenkins/plugins changed or something?